xbout/boutdataset.py

```diff
@@ -209,10 +212,12 @@ def animate_list(self, variables, animate_over='t', save_as=None, show=False, fp
         """
 
         nvars = len(variables)
+        std_format = False
```

Comment:
I think it makes sense to remove the blank figures even when nrows and/or ncols are given - I think it's unlikely that the blank axes are useful.